.circleci/config.yml

```diff
@@ -292,6 +291,34 @@ jobs:
           qualifier: "<< parameters.qualifier >>"
           releaseChannel: "<< parameters.releaseChannel >>"
 
+  clean_up_dev_deployments:
+    executor:
+      name: hmpps/node
+      tag: << pipeline.parameters.node-version >>
+    steps:
+      - checkout
+      - hmpps/k8s_setup:
+          ca-cert: KUBE_CLUSTER_CERT
+          cluster-name: KUBE_CLUSTER_NAME
+          kube-namespace: KUBE_NAMESPACE
+          kube-token: KUBE_TOKEN
+      - hmpps/install_helm
+      - run:
+          name: Clean up dev deployments
+          command: |
+            curl -s "https://api.github.com/repos/ministryofjustice/prisoner-content-hub-frontend/pulls" | jq  '.[].number' | sort > prs.txt
+            helm --namespace prisoner-content-hub-development list | awk '{ print $1 }' | tail -n +2 | grep 'frontend-' | sed "s/[^0-9]//g" | sort >> deployments.txt
+
+            deployments_to_delete=($(comm -23 deployments.txt prs.txt))
```

Comment:
this uses [comm](https://linux.die.net/man/1/comm) to create an array of all the things that are in deployments.txt and not in prs.txt